operator/v1/types_network.go

```diff
@@ -91,8 +91,8 @@ type NetworkSpec struct {
 	// deployKubeProxy specifies whether or not a standalone kube-proxy should
 	// be deployed by the operator. Some network providers include kube-proxy
 	// or similar functionality. If unset, the plugin will attempt to select
-	// the correct value, which is false when OpenShift SDN and ovn-kubernetes are
-	// used and true otherwise.
+	// the correct value, which is false when ovn-kubernetes is used and true
```

Comment:
While we are here should we change `ovn-kubernetes` to `OVNKubernetes`?

Reply:
`OVNKubernetes` with no hyphen and no space is generally only used as a value in the API, whereas we're talking about the plugin itself here really, so `ovn-kubernetes` (or `OVN-Kubernetes` or `OVN Kubernetes` seems correct)

Reply:
I think the name BU prefers in all documentation is `OVN Kubernetes`. cc @mcurry-rh.

Reply:
The existing API documentation is inconsistent and this does not change it, so that can be fixed later.